op-e2e/setup.go

```diff
@@ -178,135 +236,78 @@ func (cfg SystemConfig) start() (*System, error) {
 		}
 	}()
 
-	// Wallet
-	wallet, err := hdwallet.NewFromMnemonic(cfg.Mnemonic)
-	if err != nil {
-		return nil, fmt.Errorf("Failed to create wallet: %w", err)
-	}
-	sys.wallet = wallet
-
-	// Create the BSS and set it's config here because it needs to be derived from the accounts
-	bssPrivKey, err := wallet.PrivateKey(accounts.Account{
-		URL: accounts.URL{
-			Path: cfg.BatchSubmitterHDPath,
-		},
-	})
+	l1Genesis, err := genesis.BuildL1DeveloperGenesis(cfg.DeployConfig)
 	if err != nil {
 		return nil, err
 	}
-	batchSubmitterAddr := crypto.PubkeyToAddress(bssPrivKey.PublicKey)
 
-	p2pSignerPrivKey, err := wallet.PrivateKey(accounts.Account{
-		URL: accounts.URL{
-			Path: cfg.P2PSignerHDPath,
-		},
-	})
-	if err != nil {
-		return nil, err
+	for addr, amount := range cfg.Premine {
```

Comment:
Is this premine still necessary? Can we dedupe this code and add something to the chain ops package and not set anything here?

Reply:
It is - we sometimes want to override balances in the dev accounts, so having a way to do that without specifying all of them is nice.

We can certainly upgrade the chain ops package, but I'd rather minimize the diff here and do that separately.